build.yaml

```diff
@@ -31,4 +31,6 @@ targets:
           - test/**/*.dart
       stormberry|runner:
         generate_for:
-          - test/**/*.dart
\ No newline at end of file
+          - test/**/*.dart
+      build_test:test_bootstrap:
```

Comment:
The `build_test` package generates code to test code in all platforms, including web, but obviously those tests will fail since stormberry does not support web. Disabling the package avoids this problem